### ISSUE TITLE
Review Costs & Challenge Fees for Payment Tasks in Direct

### ISSUE THREAD

**Issue by rsial2** [appirio-tech/direct-app#207]
1. Create a F2F challenge. 
2. Set it as a task.
3. The review cost will appear even though there is no reviewer.
4. The review cost will not appear in reporting - it's not being logged anywhere, so it's just a UI issue.
5. The challenge fee is including the review payment in it's calculation, but there is no actual review payment.

**Comment by skyhit** [comment]
@rsial2 yes, that is just review cost estimation, if needed we can update the logic for review estimation calculation

**Comment by rsial2** [comment]
I think we need to update it - but it's up to @ajefts

**Comment by ajefts** [comment]
@skyhit Yes, let's do it.  If there is no reviewer on the task then let's remove the review cost estimate.  The key is that it doesn't get factored into the admin fee.  However, if a review is added, we need to make sure the review cost IS factored into the fee.

**Comment by rsial2** [comment]
So if it's set to community = add fee

If it's set to internal = no fee / no cost should appear

**Comment by skyhit** [comment]
@rsial2 @ajefts if the F2F challenge is not task, we still need to show the review cost as one iterative reviewer, right?

**Comment by ajefts** [comment]
Only if it's set to community review.  If it's an "internal review", then let's not include the estimate.  If it's something we end up paying for in an "internal review" scenario, then we can add that cost manually in OR for now.
